src/Core/Form/ChoiceProvider/LanguageByIdChoiceProvider.php

```diff
@@ -52,11 +54,12 @@ public function __construct(LanguageDataProvider $languageDataProvider)
      *
      * @return array
      */
-    public function getChoices()
+    public function getChoices(array $options = [])
     {
         $choices = [];
+        $shopId = !empty($options['shop_id']) ? $options['shop_id'] : false;
```

Comment:
I should have done this actually:

```suggestion
        $shopId = !empty((int) $options['shop_id']) ? $options['shop_id'] : false;
```